server/lib/enrich/article.js

```diff
@@ -31,6 +31,13 @@ module.exports = exports = function article(content, format) {
 		content.bodyHTML = content.body;
 	}
 
+	content.hasGraphics = content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0 ? true : false;
+
+	const countOfGraphics = content.contentStats && content.contentStats.graphics && content.contentStats.graphics;
+	const countOfSharableGraphics = content.embeds ? content.embeds.filter(embed => embed && embed.type.endsWith('Graphic')).filter(embed => embed.canBeSyndicated === 'yes').length : 0;
+
+	content.canAllGraphicsBeSyndicated = countOfGraphics > 0 ? countOfSharableGraphics < countOfGraphics : false;
```

Comment:
Decided to go with `false` rather than `null` because that way I can have `expect a boolean` in tests.